deckhouse-controller/crds/doc-ru-module-pull-override.yaml

```
@@ -31,7 +31,7 @@ spec:
                   description: "Контрольная сумма образа контейнера."
```

```suggestion
                  description: Контрольная сумма образа контейнера.
```

quotation marks in descriptions are sometimes present, sometimes not.